src/Raven.Client/Documents/Conventions/DocumentConventions.cs

```diff
@@ -193,6 +193,13 @@ public int MaxNumberOfRequestsPerSession
             }
         }
 
+        /// <summary>
+        ///     Gets or sets the max size of the cache in requestExecutor.
+        ///     Default value is 512MB
+        /// </summary>
+        /// <value>The max size of cache in requestExecutor.</value>
+        public long MaxHttpCacheSize { get; set; } = 1024 * 1024L * 512L;
```

Comment:
Maybe we can use https://github.com/ravendb/ravendb/blob/v4.0/src/Sparrow/Size.cs here? It is easier to understand for user what kind of units needs to be used. E.g. in the description here we have 512MB, so it gives me an impression that the unit here is in MB, which will cause confusion, because putting 1024 will not give me 1GB.

Reply:
I like that idea, yes.